source/include/ExecutionContext.h

```diff
@@ -63,8 +63,13 @@ enum ExecutionState
     kExecutionState_None = 0,
     kExecutionState_ClumpsInRunQueue = 0x01,
     kExecutionState_ClumpsWaitingOnTime = 0x02,
-    kExecutionState_ClumpsWaitingOnQueues = 0x04,
-    kExecutionState_ClumpsWaitingOnISRs = 0x08,
+};
+
+enum ExecSlicesResult {
```

Comment:
I don't have a lot of C++ experience, but can you define ExecSlicesResult as a union of an enum and an Int32 so that ExecuteSlices can actually return a value of that type?

Otherwise it seems strange to have the enum defined if it is not actually used anywhere, maybe it should just be commented out?

Reply:
I hear you, but unfortunately, there's no good solution to this in C++.  Normally you might create a small wrapper class for it, and make the known-values predicates, but that would be overkill here and may not be interoperable with JS as a return value.